spec/controllers/estimates_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+require 'pry'
+require 'curl'
+require 'support/params_hash.rb'
+
+RSpec.describe EstimatesController, type: :controller do
+  describe "POST 'quote'" do
+    include_context "using shipping params"
+
+    context "successful api call" do
+      it "is successful" do
+        get :quote, params, { format: :json }
+        expect(response.response_code).to eq 200
+      end
```

Comment:
To be more precise, your "it" statement should be "returns a http status code of 200".